src/main/java/org/cryptomator/linux/keychain/SecretServiceKeychainAccess.java

```diff
@@ -18,7 +18,7 @@ public boolean isSupported() {
 		try (@SuppressWarnings("unused") SimpleCollection keyring = new SimpleCollection()) {
 			// seems like we're able to access the keyring.
 			return true;
-		} catch (IOException | RuntimeException e) {
+		} catch (IOException | ExceptionInInitializerError | RuntimeException e) {
```

Comment:
Why did you add the `ExceptionInInitializerError` here? I think secret-service throws an `IOException` if something goes wrong?

Reply:
That's based upon the actual code running on my setup. With the Session DBus non-existent it threw an ExceptionInInitializerError upon running, so I just added it to the list so it marks it as not supported. Without that, it crashes similarly to the original issue, but with SecretService in the backtrace.